webui/react/src/App.tsx

```diff
@@ -60,7 +60,7 @@ const AppView: React.FC = () => {
   }, [ infoResponse, setInfo ]);
 
   useEffect(() => {
-    setupAnalytics(info);
+    setupAnalytics(info).then();
```

Comment:
**question:** what's the reason for the `then()`? If you want it to block before running the code below, you may have to move the code below into the `then`. Another approach is to create a local async function inside the `useEffect` (since you can convert the useEffect function itself into async) and then call it from `useEffect`.

Reply:
That's just to make explicit that the promise is deliberately ignored. But on this project can be skipped, I got a habit on another project where linting was complaining about that.

<img width="472" alt="Schermata 2020-12-19 alle 09 35 59" src="https://user-images.githubusercontent.com/5413702/102685026-c5615400-41dd-11eb-830c-4ae1fe93eeff.png">